docker/docker-compose-integration-test.yml

```diff
@@ -34,6 +36,6 @@ services:
       - --rollup.coinbase=f39fd6e51aad88f6f4ce6ab8827279cfffb92266
       - --rollup.l1endpoint=ws://hardhat:8545
       - --rollup.l1chainid=31337
-      - --rollup.sequencer-inbox-addr=0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
-      - --rollup.rollup-addr=0x5FC8d32690cc91D4c39d9d3abcBD16989F875707
+      - --rollup.sequencer-inbox-addr=0x2E983A1Ba5e8b38AAAeC4B440B9dDcFBf72E15d1
+      - --rollup.rollup-addr=0xF6168876932289D073567f347121A267095f3DD6
```

Comment:
can you add a TODO somewhere to get rid of this hardcoding (or do it in this PR)